osf/management/commands/migrate_deleted_date.py

```diff
@@ -0,0 +1,96 @@
+import datetime
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db import connection, transaction
+from framework.celery_tasks import app as celery_app
+
+logger = logging.getLogger(__name__)
+
+TABLES_TO_POPULATE_WITH_MODIFIED = [
+    'addons_zotero_usersettings',
+    'addons_dropbox_usersettings',
+    'addons_dropbox_nodesettings',
+    'addons_figshare_nodesettings',
+    'addons_figshare_usersettings',
+    'addons_forward_nodesettings',
+    'addons_github_nodesettings',
+    'addons_github_usersettings',
+    'addons_gitlab_nodesettings',
+    'addons_gitlab_usersettings',
+    'addons_googledrive_nodesettings',
+    'addons_googledrive_usersettings',
+    'addons_mendeley_nodesettings',
+    'addons_mendeley_usersettings',
+    'addons_onedrive_nodesettings',
+    'addons_onedrive_usersettings',
+    'addons_osfstorage_nodesettings',
+    'addons_osfstorage_usersettings',
+    'addons_bitbucket_nodesettings',
+    'addons_bitbucket_usersettings',
+    'addons_owncloud_nodesettings',
+    'addons_box_nodesettings',
+    'addons_owncloud_usersettings',
+    'addons_box_usersettings',
+    'addons_dataverse_nodesettings',
+    'addons_dataverse_usersettings',
+    'addons_s3_nodesettings',
+    'addons_s3_usersettings',
+    'addons_twofactor_usersettings',
+    'addons_wiki_nodesettings',
+    'addons_zotero_nodesettings'
+]
+
+UPDATE_DELETED_WITH_MODIFIED = 'SET statement_timeout = 10000; UPDATE {} SET deleted=modified WHERE id IN (SELECT id FROM {} WHERE is_deleted AND deleted IS NULL LIMIT {}) RETURNING id;'
+
+
+@celery_app.task(name='management.commands.migrate_deleted_date')
+def populate_deleted(dry_run=False, page_size=1000):
+    with transaction.atomic():
+        for table in TABLES_TO_POPULATE_WITH_MODIFIED:
+            run_statements(UPDATE_DELETED_WITH_MODIFIED, page_size, table)
+        if dry_run:
+            raise RuntimeError('Dry Run -- Transaction rolled back')
+
+def run_statements(statement, page_size, table):
+    logger.info('Populating deleted column in table {}'.format(table))
+    with connection.cursor() as cursor:
+        cursor.execute(statement.format(table, table, page_size))
+        rows = cursor.fetchall()
+        if not rows:
+            logger.info('Sentry notification that {} is populated'.format(table))
```

Comment:
Unlike your other `deleted` PR - this is going to take much fewer runs.  It is difficult to sort through the output to see if all the tables are populated, because there are so many tables, and each run attempts to populate tables and continues to log that sentry was notified, even if it's already been notified before.

For this one, since the majority will be populated on the first run (if we have a 1000 row default size) - so instead of `not rows` I'd log if `rows` - (table) may still have rows that need to be populated.  Something easy to display tables remaining instead of tables completed.